src/pages/en/guides/content-collections.md

```diff
@@ -0,0 +1,363 @@
+
+Content within a collection should share the same frontmatter shape and types. You can optionally enforce these types [by configuring a schema](/en/guides/content-collections/#defining-a-collection-schema).
+
+To create a collection, add a new directory to `src/content/`. Then, add Markdown or MDX entries that share frontmatter properties. The following example shows two collections: `blog` and `newsletter`. 
+
+```bash "newsletter/" "blog/"
+src/content/
+│   # All blog posts have the same frontmatter properties
+├── blog/
+│   ├── columbia.md
+│   ├── endeavour.md
+│   └── enterprise.md
+│   # All newsletters have the same frontmatter properties
+└── newsletter/
+    ├── week-1.md
+    ├── week-2.md
+    └── week-3.md
+```
+
+### Collections with nested directories
+
+Collections are **top-level folders** within `src/content/`. You cannot nest collections, but you may use nested directories within a collection to better organize a collection's content. All nested directories will share the same schema defined for the top-level collection.
+
+For example, you can use this structure for internationalization:
+
+```bash
+src/content/
+└── docs/
+    │   # docs schema applies to all nested directories 👇
+    ├── en/
+    ├── es/
+    └── ...
+```
+
+
+## Defining a collection schema
+
+Schemas are an optional way to enforce frontmatter types in a collection. Astro uses [Zod](https://github.com/colinhacks/zod) to validate your frontmatter with schemas in the form of [Zod objects](https://github.com/colinhacks/zod#objects).
+
+To configure schemas, create a `src/content/config.ts` file (`.js` and `.mjs` extensions are also supported). This file should:
+
+1. Import the `defineCollection` and `z` utilities from `astro:content`. 
+2. Define a `schema` for each collection.
+2. Export a single `collections` object, with each object key corresponding to the collection's folder name.
+
+For example, say you maintain two collections: one for release announcements and one for blog content. Your entries at `src/content/announcements` should include a `title` and `version`.  Your `src/content/engineering-blog/` collection entries should have a `title`, list of `tags`, and an optional `image` URL. 
+
+You can specify each expected property in the `schema` field of `defineCollection`:
+
+```ts
+// src/content/config.ts
+import { z, defineCollection } from 'astro:content';
+
+const releases = defineCollection({
+  schema: {
+    title: z.string(),
+    version: z.number(),
+  },
+});
+
+const engineeringBlog = defineCollection({
+  schema: {
+    title: z.string(),
+    tags: z.array(z.string()),
+    image: z.string().optional(),
+  },
+});
+
+export const collections = {
+	releases: releases,
```

Comment:
Er… I guess this is left over from some old draft review 😅 Not wrong though!

Reply:
Ooh, I did want to bring up something from @bholmesdev stream here:

On the stream, this function was:

```
export const collections = {
	blog,
	cars,
}
```

Do we want to use `releases,` (instead of `releases: realeases,`)? What is going to be our recommended example in this case, where we don't have to rename the collection?

Reply:
Definitely `releases` over `releases: releases`!

Reply:
So, in personal practice I would _always_ use the shorthand `{ releases }`. BUT in docs examples I tend to prefer the longhand just in case some users aren’t familiar with how the shorthand works and aren’t sure how to modify the example given their namings. But it’s a toss up basically.